backend/src/rules/create_game.clj

```diff
@@ -2,22 +2,35 @@
   (:require [configs.hands :as hands]
             [configs.rows :as rows]))
 
-(defn ^:private new-player
+(defn new-player
   "Creates a new player object"
-  ([hand]
-   {
-    :hand hand
-    }))
+[hand player]
+{
+ :hand (loop [final-hand []
```

Comment:
```clojure
(mapv (fn [card id] (assoc card :id id)) hand (iterate inc 1000))
```

Reply:
I don't think this does the same as the loop.
The loop is supposed to use indexes begining with 0 for player 0, and beginning with 1000 for player 1.